parse_test.go

```diff
@@ -0,0 +1,41 @@
+package tea
+
+import (
+	"image/color"
+	"reflect"
+	"testing"
+)
+
+func TestParseSequence_Events(t *testing.T) {
+	input := []byte("\x1b\x1b[Ztest\x00\x1b]10;rgb:1234/1234/1234\x07\x1b[27;2;27~\x1b[?1049;2$y")
```

Comment:
should we maybe be generating the input here programmatically? If this test fails, it seems like it might be a bit of a headache to figure out what went wrong with this sequence

Reply:
Always good to have more tests. We already do for key sequences in `key_test.go`.

Reply:
I've added more tests for Kitty/fixterms, Xterm modifyOtherKeys, and terminal bg/fg/cur colors in their respective PRs.